res/sectors.xsd

```diff
@@ -73,7 +73,7 @@
           <xs:element name="Subsectors">
             <xs:complexType>
               <xs:sequence>
-                <xs:element maxOccurs="unbounded" name="Subsector">
+                <xs:element minOccurs="0" maxOccurs="16" name="Subsector">
```

Comment:
The min occurances is to allow for the fact the xml returned by the metadata API can have an empty <Sectors/> element.

Setting the max occurances to 16 seems logical.